scripts/squads.yml

```diff
@@ -0,0 +1,216 @@
+    github: inamiy
+    twitter: inamiy
+  - &James
+    name: James Birtwell
+    github: jimmybee
+  - &Joshua
+    name: Joshua Simmons
+    github: j531
+  - &Julien
+    name: Julien Ducret
+    github: brocoo
+  - &Konrad
+    name: Konrad Muchowicz
+    github: konrad-em
+  - &Martin
+    name: Martin Nygren
+    github: zzcgumn
+  - &Michael
+    name: Michael Brown
+    github: mluisbrown
+    twitter: mluisbrown
+  - &Michal
+    name: Michał Kwiecień
+    github: MichalTKwiecien
+    twitter: kwiecien_co
+  - &Olivier
+    name:  Olivier Halligon
+    github: AliSoftware
+    twitter: aligatr
+  - &Patrick
+    name: Patrick Westmeyer
+    github: bh-pwestmeyer
+  - &Rui
+    name: Rui Peres
+    github: RuiAAPeres
+    twitter: peres
+  - &Sam
+    name: Sam Francis
+    github: SamFrancis-Babylon
+  - &Sergey
+    name: Sergey Shulga
+    github: sergdort
+    twitter: SergDort
+  - &Simon
+    name: Simon Cass
+    github: scass91
+    twitter: codercass
+  - &Viorel
+    name: Viorel Mihalache
+    github: viorel15
+    twitter: viorelMO
+  - &Witold
+    name: Witold Skibniewski
+    github: mr-v
+  - &Yuri
+    name: Yuri Karabatov
+    github: karabatov
+    twitter: karabatov
+
+
+################################################################
+# Global roles transverse to the squads
+#
+roles:
+  - name: Engineering Manager
+    members:
+      - *Rui
+  - name: Chapter Leads
+    members:
+      - *Catarina
+      - *David
+  - name: Squad Tech Leads
+    members:
+      - *Michael
+
+
+################################################################
+# List of the Squads and their members
+#
+squads:
+  - name: Booking
+    desc: Face to face appointment booking.
+    members:
+      - *Witold
+  - name: Consultation
+    desc: Flows for all the consultation like prescriptions and video consultation.
+    members:
+      - *Adrian
+      - *Chitra
+  - name: Core Experience
+    desc: General user experience in the app.
+    members:
+      - *Emese
+      - *Giorgos
+      - *Sergey
+      - *Yuri
+  - name: Native Apps Platform
+    desc: Engineering work like tooling, CI and development processes.
+    members: 
+      - *David
+      - *Ilya
+      - *Inamiy
+      - *Martin
+      - *Olivier
+  - name: GP @ Hand
+    desc: End to end journey for NHS registration.
+    members: 
+      - *James
+  - name: Healthcheck
+    desc: Overview of your health using a 3D body model (avatar).
+    members: 
+      - *Ben
+      - *Julien
+  - name: Monitor
+    desc: Monitoring health metrics like activity, blood, urine, ...
+    members: 
+      - *Anders
+      - *DanielH
+      - *DanielS
+      - *Diego
+      - *Joshua
+  - name: Prescriptions
+    desc: Precriptions functionality.
+    members: 
+      - *Adam
+      - *Konrad
+  - name: Professional Services
+    members: 
+      - *Simon
+  - name: SDK
+    desc: Develop and maintain the SDK frameworks.
+    members: 
+      - *Viorel
+  - name: Tenancy & Features
```

Comment:
@anilputtabuddhi Would be nice if we had a description of this Squad like the others. We have to admit we don't really know how to describe this one 😉